pkgs/tools/networking/s4cmd/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, python3Packages }:
```

Comment:
@FRidh this application works and has support for both python2 and python3; should I set this as `pythonPackages` and let it float to that alias, or should I advertise that it works with python3 as above?

Reply:
Is there any point to running it under python2? It's fine as it is now imo.

Reply:
The key point is not what it runs now, but to show that it is compatible with both. This is/was important for the whole 2 to 3 migration. But, as that's not really happening I don't care much anymore.

Reply:
Ok, I'll leave it as `python3Packages` then, since that very clearly shows it's compatible with python3, and is unlikely to have support dropped by upstream anytime soon.

Is there an issue/RFC/ticket/PR discussing some of the 2 to 3 migration plans/strategies/thoughts?  Curious to learn more.